Classes/Util.php

```diff
@@ -120,23 +122,30 @@ public static function timestampToIso($timestamp) {
 	 * @return integer unix timestamp
 	 */
 	public static function isoToTimestamp($isoTime) {
-			// FIXME use DateTime::createFromFormat (PHP 5.3+)
-		$parsedTime = strptime($isoTime, '%Y-%m-%dT%H:%M:%SZ');
-
-		$timestamp = mktime(
-			$parsedTime['tm_hour'],
-			$parsedTime['tm_min'],
-			$parsedTime['tm_sec'],
-				// strptime returns the "Months since January (0-11)"
-				// while mktime expects the month to be a value
-				// between 1 and 12. Adding 1 to solve the problem
-			$parsedTime['tm_mon'] + 1,
-			$parsedTime['tm_mday'],
-				// strptime returns the "Years since 1900"
-			$parsedTime['tm_year'] + 1900
-		);
+		$dateTime = \DateTime::createFromFormat(self::SOLR_ISO_DATETIME_FORMAT, $isoTime);
+		return $dateTime ? (int)$dateTime->format('U') : 0;
+	}
+
+	/**
+	 * Converts a date from unix timestamp to ISO 8601 format in UTC timezone.
+	 *
+	 * @param	integer	$timestamp unix timestamp
+	 * @return	string	the date in ISO 8601 format
+	 */
+	public static function timestampToUtcIso($timestamp) {
```

Comment:
https://github.com/TYPO3-Solr/ext-solr/pull/8/files#diff-99a1d1c23fd6b626eea8167b0dd97934R115

It is there?